boards/silabs/efr32_radio/efr32_radio-pinctrl.dtsi

```diff
@@ -16,4 +16,30 @@
 				<GECKO_LOC(UART_RX, 0)>;
 		};
 	};
+
+	usart1_default: usart1_default {
+		group1 {
+			psels = <GECKO_PSEL(SPIM_SCK, C, 8)>,
+				<GECKO_PSEL(SPIM_MISO, C, 7)>,
+				<GECKO_PSEL(SPIM_MOSI, C, 6)>,
+				<GECKO_PSEL(SPIM_CS, A, 4)>,
```

Comment:
Selecting a pin for CS in pinctrl gives the USART peripheral hardware control of the pin. This isn't compatible with  the use of `cs-gpios` in `&usart1` and `&usart2`, which indicates software/GPIO control of the pin. As far as I can tell, the spi_gecko driver also doesn't configure AUTOCS as part of init, so even if the USART is given hardware control through pinctrl, it won't actually drive the pin.

I'd suggest removing SPIM_CS from the pinctrl setup to make it functionally equivalent to the previous implementation.